src/hooks/__tests__/useGetProfileArray.test.ts

```diff
@@ -62,11 +62,11 @@ describe('useGetProfileArray', () => {
             setProfile: () => {},
         });
         vi.mocked(getProfileArray).mockResolvedValue(['movie-123', 'tv-456']);
-        const { result } = renderHook(({ whichCol }) => useGetProfileArray(whichCol), {
-            initialProps: { whichCol: WHICH_COL[0] },
+        const { result } = renderHook(({ data: { whichCol } }) => useGetProfileArray(whichCol), {
+            initialProps: { data: { whichCol: WHICH_COL[0] } },
```

Comment:
```suggestion
        const { result } = renderHook(({ whichCol }) => useGetProfileArray(whichCol), {
            initialProps: { whichCol: WHICH_COL[0] },
```
This should remain unchanged as the props provided to the hook are the same. Only the return value has changed.

Reply:
@Thenlie Thanks for this catch. Took me a minute to understand this function and I modified the file pretty heavily so I didn't have the original setup. Repushed